CHANGELOG

```diff
@@ -11,7 +11,8 @@ Current head v.2.1.0 rc
 - CMake
   * Remove dead code (#15)
 - Licensing
-  * Added license and readme to externals (#18)
```

Comment:
Sounds weird, what happened to that line ?

Reply:
We have to use the present or the past tense, but remain consistent in the all file.

Reply:
is it possible to change commit already merged ?

Reply:
Directly from main

Reply:
I can remove the modification here.
But we should have naming convention in a CHANGELOG file, are you agree with that ?

Reply:
I revert the change now, check CHANGELOG